### ISSUE TITLE
nginx: exited - code 1 - Need help setting up.

### ISSUE THREAD

**Issue by hendrikhaehnel** [indico/indico-containers#61]
Hi there,
I am trying to get indico running since my University does not provide it. 
I set up an LXC container on Proxmox based on Ubuntu 24.10 (root user). Running portainer, but used the "docker compose up" instead of defining the stack in Portainer (wanted to be as close to the documented way as possible)

I tried running the docker-compose with minimal changes to the configuration (just set the secret key as required). 

Something is not working, the nginx container is exiting (exit - code 1) with the following error:
![image](https://github.com/user-attachments/assets/70ec440c-d455-4985-97c9-c19db98174be)

And the output of docker compose up is stuck trying to connect to the DB.
![image](https://github.com/user-attachments/assets/fdf373c5-5696-486e-ad9f-7e550546dfbc)

Any pointers? I am hoping, I just missed some simple config.

**Comment by tomasr8** [comment]
Hi, can you post the whole docker compose output?

**Comment by hendrikhaehnel** [comment]
I fixed it ...
1. The random string generated by the proposed script (ran it with python3) seemed to produce some characters that broke something. I just put some random string in there without special characters and everything seemed fine. 
2. nginx still exited the first time I ran compose, but it was fixed by manually restarting the nginx docker. 

So for now I am happy ... lets see. Thank you anyways
